epub33/rs/index.html

```diff
@@ -599,19 +599,19 @@ <h3>MathML</h3>
 
 						<ul class="conformance-list">
 							<li>
-								<p id="confreq-mathml-rs-behavior">MUST be an input-compliant renderer for <a
+								<p id="confreq-mathml-rs-behavior">MUST process <a
```

Comment:
There are three types of compliant processors per chapter 7 of that specification: input-compliant, output-compliant, and roundtrip-compliant. See https://www.w3.org/TR/REC-MathML/chapter7.html#sec7.2.1

It somewhat stands to reason that a reading system only has to be input-compliant, but unless we can inherit processing from HTML instead of directly from MathML (and not need to say anything here) this change adds ambiguity to what we mean. Unfortunately, HTML still seems a bit open-ended about support, even if input-compliant presentational math processing is all we'll probably ever see supported.

Adding a link to the original wording would certainly help, though.

Reply:
OK, how about "input-compliant processor" (language used there) along with a link? Then "render" only applies to visual RSes. (I recognize that not all rendering is visual; for example, we refer to audio rendering elsewhere. Nonetheless, I think "renderer" is unnecessary here.)

Reply:
LGTM!